### PR TITLE
start: clarify Data Mgmt/ Plots section

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Follow-up to https://github.com/iterative/dvc.org/pull/4145#pullrequestreview-1228507276

In review app: https://dvc-org-start-plots-leoi61spbm.herokuapp.com/doc/start/data-management/metrics-parameters-plots

### CODE REVIEW

File: content/docs/start/data-management/metrics-parameters-plots.md
Comment (jorgeorpinel) [inline content/docs/start/data-management/metrics-parameters-plots.md:124]: Which one is shown in the JSON sample below?

File: content/docs/start/data-management/metrics-parameters-plots.md
Comment (jorgeorpinel) [inline content/docs/start/data-management/metrics-parameters-plots.md:133]: If we could combine this paragraph with the previous one (before the code block), the section would be more readable: just a p and a code block before we get to the main point (configuring and rendering plots).

- [ ] I'll try ⏳

File: content/docs/start/data-management/metrics-parameters-plots.md
Comment (dberenbaum) [inline content/docs/start/data-management/metrics-parameters-plots.md:87]: 2 questions:
1. Do we mention train and test datasets above? How will someone know to what "both datasets" refers?
2. What's the rationale for removing links to background info on those metrics?

File: content/docs/start/data-management/metrics-parameters-plots.md
Comment (dberenbaum) [inline content/docs/start/data-management/metrics-parameters-plots.md:119]: Let's move this into the following paragraph per https://github.com/iterative/dvc.org/pull/4145#discussion_r1056031907 🙏

File: content/docs/start/data-management/metrics-parameters-plots.md
Comment (dberenbaum) [inline content/docs/start/data-management/metrics-parameters-plots.md:130]: This is not the actual file structure, though. Why change it? I worry users could try to generate a file with that structure and be surprised it doesn't work as expected.

File: content/docs/start/data-management/metrics-parameters-plots.md
Comment (dberenbaum) [inline content/docs/start/data-management/metrics-parameters-plots.md:134]: Do we need to clarify that this is part of `dvc.yaml`? Why introduce the term "rendering properties"?

More generally (but minor), why so much use of "rendering" in this PR? I don't have strong opinion on it, but I'm not sure it's more clear than what it's replacing.

**Review by dberenbaum — CHANGES_REQUESTED** [review]
Thanks for taking a look @jorgeorpinel! I agree it's worth it since the get started is so important, but it's not clear to me how some of the changes make meaningful improvements.

File: content/docs/start/data-management/metrics-parameters-plots.md
Comment (jorgeorpinel) [reply content/docs/start/data-management/metrics-parameters-plots.md:87]: The 2 sets are shown in the example itself but I removed that parenthesis anyway (not needed). The metrics are still linked, to the specific framework implementation docs. Those docs link to more background info, so having 2 sets of different links for metrics here seemed unnecessary.

**Comment by dberenbaum** [comment]
@jorgeorpinel Thanks for updating! Are you waiting on another review?

File: content/docs/start/data-management/metrics-parameters-plots.md
Comment (jorgeorpinel) [reply content/docs/start/data-management/metrics-parameters-plots.md]: Done.

File: content/docs/start/data-management/metrics-parameters-plots.md
Comment (jorgeorpinel) [reply content/docs/start/data-management/metrics-parameters-plots.md]: I thought it was easier to read since we were referring to the JSON array above but you're right. Updated, PTAL.

File: content/docs/start/data-management/metrics-parameters-plots.md
Comment (jorgeorpinel) [reply content/docs/start/data-management/metrics-parameters-plots.md]: Changed to "configure the plots in dvc.yaml". Changed other instances of "render" to "visualizing" and "generate", for consistency with existing docs terminology (good catch).

**Comment by jorgeorpinel** [comment]
> not clear to me how some of the changes make meaningful improvements.

@dberenbaum we fight for every word in Get Started pages 🙂 the changes mainly consolidate paragraphs, improve readability, and rename [this section](https://dvc-org-start-plots-leoi61spbm.herokuapp.com/doc/start/data-management/metrics-parameters-plots#visualizing-plots).

File: content/docs/start/data-management/metrics-parameters-plots.md
Comment (dberenbaum) [reply content/docs/start/data-management/metrics-parameters-plots.md]: Okay, I missed that the background info was linked in the metrics already. Agreed that it's not needed in both places, plus to me it looks like these plots links should go to the actual files in the repo, so I'm good with removing them.

File: content/docs/start/data-management/metrics-parameters-plots.md
Comment (dberenbaum) [reply content/docs/start/data-management/metrics-parameters-plots.md]: This is the one part that still doesn't read well to me. Per https://github.com/iterative/dvc.org/pull/4145#discussion_r1056031907, I thought the motivation was to mention all the plots files in this paragraph so that the next paragraph went straight to configuring them, but there is still a hanging paragraph about the feature importance image file. 

Also, I find the structure/order confusing. It starts with the roc and confusion matrix without any explanation of what those files look like, then gives a detailed explanation and example for precision-recall (seems like there must be something unique about this file for it to be listed separately with so much detail, but it's not clear why), and finally mentions a custom image after the example.

What about something like this?


> The stage also writes [plots files](/doc/user-guide/project-structure/dvcyaml-files#plots) you can visualize:
> 
> * [DVCLive](/doc/dvclive)-generated ROC curves and confusion matrices under `eval/live`.
> * Custom precision-recall curves as JSON arrays (format shown below) under `eval/prc`.
> * Custom image showing a bar chart of features' importance in `eval/importance.png`.
>
> ```
> {
>   "prc": [
>     { "precision": 0.0214, "recall": 1.0, "threshold": 0.0 },
>     ...
>     { "precision": 1.0, "recall": 0.0093, "threshold": 0.6 }
>   ]
> }
> ```

It puts everything in one paragraph with a uniform structure (💅 I also changed the JSON example back but truncated the values to focus on it being an array rather than the fields within each element of the array).

File: content/docs/start/data-management/metrics-parameters-plots.md
Comment (jorgeorpinel) [reply content/docs/start/data-management/metrics-parameters-plots.md]: I tried linking the file path to the repo but it's too much (too many links in that sentence already and since the code block below is the same). But feel free to revise the linking of that and other paragraphs that also mention file paths in the repo if you prefer.

File: content/docs/start/data-management/metrics-parameters-plots.md
Comment (jorgeorpinel) [reply content/docs/start/data-management/metrics-parameters-plots.md]: I did try to mention all in a single paragraph as originally suggested, but it was hard to read and unclear why the code sample below only focused on one of them. I like your idea though! I applied it but put the JSON example under that specific bullet. PTAL

Comment (dberenbaum) [reply content/docs/start/data-management/metrics-parameters-plots.md]: One last thing: now that we have this paragraph more structured, it stick out to me that the `dvc.yaml` below it is in a different order. Should we rearrange that `dvc.yaml` to match the order of the bullets? Not a blocker, but I think it would be more readable.

**Review by dberenbaum — APPROVED** [review]
Thanks again @jorgeorpinel!

File: content/docs/start/data-management/metrics-parameters-plots.md
Comment (jorgeorpinel) [reply content/docs/start/data-management/metrics-parameters-plots.md]: Yeah I had also noticed that it's the exact inverse order but thought that's just how the real file will look file to the user so left it like that. If we change it here would have to modify the sample repo prob? If we do we can revisit this any time @dberenbaum

Comment (jorgeorpinel) [reply content/docs/start/data-management/metrics-parameters-plots.md]: Yup looks like it's a copy/paste ATM:

https://github.com/iterative/example-get-started/blob/7e4e86ca117f1bbef288f2abebfc7c97d0a9925d/dvc.yaml#L48-L65

Comment (dberenbaum) [reply content/docs/start/data-management/metrics-parameters-plots.md]: Yeah, there's no functional difference, but we can reorder it in both places to match the bullets here.

**Pull request merged**
